docs/.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.1
```

Comment:
What's this (and the webrick dependency below) coming from?  I'm surprised editing doc introduces a new dependency, and it looks like webrick is an HTTP server?

Reply:
is there such a thing as dev dependencies in there then?  (make it clear it's local dev stuff not something we need at runtime).